app/controllers/search_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SearchController < ApplicationController
+  before_action :authenticate_user!
+  include ApplicationHelper
+
+  def index
+    @search_query = params.permit(:query)["query"]
```

Comment:
we don't need to `.permit(:query)`, we can just do `params[:query]` here

Reply:
Ah - sorry... that was me trying to get past the "turbolinks won't let me update the canvas" issue I was having.  I tried a _lot_ of stuff that didn't work :(